tests/sqlparser_common.rs

```diff
@@ -11460,3 +11460,54 @@ fn test_try_convert() {
         all_dialects_where(|d| d.supports_try_convert() && !d.convert_type_before_value());
     dialects.verified_expr("TRY_CONVERT('foo', VARCHAR(MAX))");
 }
+
+#[test]
+fn parse_bang_not() {
+    let dialects = all_dialects_where(|d| d.supports_bang_not_operator());
+    let sql = "SELECT !a, !(b > 3)";
+    let Select { projection, .. } = dialects.verified_only_select(sql);
+
+    for (i, (op, expr)) in [
```

Comment:
the `op` variable seems redundant given its always set to `UnaryOperator::BangNot`? so that the loop can probably be simplified to `for (i, expr) in   [...]`

Reply:
Good idea. Have Done!